Wangscape/tilegen/alpha/CalculatorTopTwo.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <utility>
+#include <vector>
+
+#include "CalculatorBase.h"
+
+
+namespace tilegen
+{
+namespace alpha
+{
+
+
+class CalculatorTopTwo :
+    public tilegen::alpha::CalculatorBase
+{
+public:
+    double power = 1.;
+protected:
+    virtual void updateAlphasAux(const Weights& weights);
```

Comment:
Is that really necessary? `CalculatorBase::updateAlphasAux` is pure virtual, so `override` won't add any more safety in case of incorrect declarations.

#100's `CalculatorDither::updateAlphasAux` could use an `override` specifier, though.